Multisig/Logic/Models/Address.swift

```diff
@@ -68,6 +68,10 @@ struct Address: Hashable, ExpressibleByStringInterpolation, CustomStringConverti
         self == .zero
     }
 
+    var isEmpty: Bool {
+        isZero || checksummed.isEmpty || checksummed == "0"
```

Comment:
is it possible that the checksummed is empty or "0"?

Reply:
Actually this condition I moved from AdvancedSafeSettingsView.swift as it is, so I assumed that it's possible, doesn't it?

Reply:
the `checksummed` is a derived property from the binary data, so whenever the Data is 0, the checksummed will always be the `0x000000......00`, so imo this is not possible - but we can check it with unit test 😉 

In that case, this isEmpty is not needed at all

Reply:
Still we have the case that checksummed is empty text right ?